components/CounterForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useCart } from "../context/ItemsInCart";
+
+const CounterForm = ({ slug, name, isTiny }) => {
+  const { itemsInCart, setItemsInCart, addItem } = useCart();
+  const [amount, setAmount] = useState(0);
+
+  const decAmount = (int) => {
+    setAmount((amount) => amount - int);
+  };
+
+  const incAmount = (int) => {
+    setAmount((amount) => amount + int);
+  };
+
+  const handleSubmit = (event) => {
+    addItem({ name, slug, amount: amount });
+    event.preventDefault();
+    setAmount(() => 0);
+  };
+
+  const amountHandlerInputChange = (event) => {
+    setAmount(parseInt(event.target.value));
+    console.log(typeof amount);
+  };
+
+  /* 
+    - Input + 
+    Submit
+    
+    */
+
+  return (
+    <StyledForm onSubmit={handleSubmit} isTiny>
+      <fieldset>
+        <input type="button" onClick={() => decAmount(1)} value="-" />
```

Comment:
For the plus/minus buttons, it might make sense to annotate what the buttons do to a screenreader, since the - or + are not very self explanatory.

You can use `aria-label="decrement amount by 1"`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute#examples